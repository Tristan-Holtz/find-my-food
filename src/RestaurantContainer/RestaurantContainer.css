.restaurant-container {
  background-color: gray;
  box-shadow: inset 0 5px 10px #000000;
  margin-top: 13.5em;
  overflow: auto;
  width: 100%;
}

.container-top {
  background-color: gray;
  display: flex;
  justify-content: space-between;
  margin-top: 3.5em;
  padding: 1em;
  padding-top: 4em;
  position: fixed;
  top: 0;
  width: 97%;
  z-index: 10;
}

.page-button {
  background-color: gray;
  border: none;
  color: #fff;
  cursor: pointer;
  font-size: 1.25em;
  outline: none;
}

.page-button:hover {
  color: orange;
  font-size: 1.5em;
}

.header-div {
  background-color: orange;
  display: flex;
  flex-direction: row;
  position: fixed;
  top: 0;
  width: 100%;
  z-index: 100;
  -webkit-transition: background-color 0.25s ease-out;
}

.header {
  align-items: center;
  display: flex;
  justify-content: center;
  width: 100%;
}

.header-div:hover {
  background-color: darkorange;
}

.page-display {
  color: #fff;
}

.page-display_bottom {
  color: #fff;
  text-align: center;
  width: 100%;
}

h1 {
  color: #fff;
  margin-right: 9em;
}

h3 {
  color: #fff;
}

label {
  color: #fff;
  font-weight: bold;
}

input {
  padding: 0.1em;
}

body {
  background-color: gray;
}
