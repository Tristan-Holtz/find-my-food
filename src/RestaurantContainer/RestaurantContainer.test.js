import React from "react";
import { RestaurantContainer } from "./RestaurantContainer";
import { shallow } from "enzyme";

describe("RestaurantContainer", () => {
  let wrapper;
  let props = {
    restaurants: [1, 2, 3],
    currentPage: [1],
    currentPages: [1, 2],
  };
  it("should render the component", () => {
    wrapper = shallow(<RestaurantContainer {...props} />);

    expect(wrapper).toMatchSnapshot();
  });
});
