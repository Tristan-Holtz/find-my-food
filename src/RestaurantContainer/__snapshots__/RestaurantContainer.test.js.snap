// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RestaurantContainer should render the component 1`] = `ShallowWrapper {}`;
